service/interface.go

```diff
@@ -0,0 +1,16 @@
+package service
+
+import (
+	"github.com/AliShan302/Task_Schedular/db"
+)
+
+// Service initializes our database instance
+type Service struct {
+	db    db.DataStore
+	Store db.DataStore
```

Comment:
what is this Store for?
are we using it anywhere?

Reply:
yeah we use it in runtime_test